plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergGlueCatalogAccessOperations.java

```diff
@@ -0,0 +1,502 @@
+        extends AbstractTestQueryFramework
+{
+    private final String schema = "test_schema_" + randomTableSuffix();
+    private final Session TEST_SESSION = testSessionBuilder()
+            .setCatalog("iceberg")
+            .setSchema(schema)
+            .build();
+
+    private GlueMetastoreStats catalogStats;
+    private GlueMetastoreStats tableOperationStats;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        File tmp = Files.createTempDirectory("test_iceberg").toFile();
+        DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(TEST_SESSION).build();
+
+        AtomicReference<GlueMetastoreStats> catalogStatsReference = new AtomicReference<>();
+        AtomicReference<GlueMetastoreStats> tableOperationStatsReference = new AtomicReference<>();
+
+        queryRunner.installPlugin(new TestingIcebergPlugin(
+                Optional.empty(),
+                Optional.empty(),
+                Optional.of(new StealStatsModule(catalogStatsReference, tableOperationStatsReference))));
+        queryRunner.createCatalog("iceberg", "iceberg",
+                ImmutableMap.of(
+                        "iceberg.catalog.type", "glue",
+                        "hive.metastore.glue.default-warehouse-dir", tmp.getAbsolutePath()));
+
+        queryRunner.execute("CREATE SCHEMA " + schema);
+
+        catalogStats = verifyNotNull(catalogStatsReference.get(), "catalogStatsReference not set");
+        tableOperationStats = verifyNotNull(tableOperationStatsReference.get(), "tableOperationStatsReference not set");
+        return queryRunner;
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void cleanUpSchema()
+    {
+        getQueryRunner().execute("DROP SCHEMA IF EXISTS " + schema);
+    }
+
+    @Test
+    public void testCreateTable()
+    {
+        try {
+            assertGlueMetastoreApiInvocations("CREATE TABLE test_create (id VARCHAR, age INT)",
+                    ImmutableMultiset.builder()
+                            .add(CREATE_TABLE)
+                            .add(GET_DATABASE)
+                            .add(GET_TABLE)
+                            .build());
+        }
+        finally {
+            getQueryRunner().execute("DROP TABLE IF EXISTS test_create");
+        }
+    }
+
+    @Test
+    public void testCreateTableAsSelect()
+    {
+        try {
+            assertGlueMetastoreApiInvocations("CREATE TABLE test_ctas AS SELECT 1 AS age",
+                    ImmutableMultiset.builder()
+                            .add(GET_DATABASE)
+                            .add(CREATE_TABLE)
+                            .add(GET_TABLE)
+                            .build());
+        }
+        finally {
+            getQueryRunner().execute("DROP TABLE IF EXISTS test_ctas");
+        }
+    }
+
+    @Test
+    public void testSelect()
+    {
+        try {
+            assertUpdate("CREATE TABLE test_select_from (id VARCHAR, age INT)");
```

Comment:
add newline (consistently with test code below)

```suggestion
            assertUpdate("CREATE TABLE test_select_from (id VARCHAR, age INT)");

```